src/test/java/com/algorand/algosdk/logic/TestLogic.java

```diff
@@ -162,7 +162,7 @@ public void testCheckProgramInvalidOpcode() throws Exception {
 
         assertThatThrownBy(() -> checkProgram(program, args))
                 .isInstanceOf(IllegalArgumentException.class)
-                .hasMessage("invalid instruction");
+                .hasMessage("invalid instruction: 129");
```

Comment:
What does 129 mean here?

Reply:
That just happens to be the invalid instruction that the program tests:
```
        byte[] program = {
            0x01, 0x20, 0x01, 0x01, (byte)0x81
        };
```
0x81 == 129